From e503164f5c7813a27390bbf1ea21b0b36d48410c Mon Sep 17 00:00:00 2001
From: Adam Farden <adam@farden.cz>
Date: Tue, 25 Oct 2016 21:43:48 +0200
Subject: [PATCH] Add msm8976 tasha sound card detection to msm8916 HAL

msm8976 is a variant of msm8952, both use the msm8916 HAL

Signed-off-by: Adam Farden <adam@farden.cz>
---
 hal/msm8916/hw_info.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/hal/msm8916/hw_info.c b/hal/msm8916/hw_info.c
index b66bec4..a4f3943 100644
--- a/hal/msm8916/hw_info.c
+++ b/hal/msm8916/hw_info.c
@@ -52,6 +52,8 @@ static void update_hardware_info_8x16(struct hardware_info *hw_info, const char
         strlcpy(hw_info->name, "msm8952", sizeof(hw_info->name));
     }  else if (!strcmp(snd_card_name, "msm8952-l9300-snd-card")) {
         strlcpy(hw_info->name, "msm8952", sizeof(hw_info->name));
+    }  else if (!strcmp(snd_card_name, "msm8976-tasha-snd-card")) {
+        strlcpy(hw_info->name, "msm8976", sizeof(hw_info->name));
     } else {
         ALOGW("%s: Not an  8x16/8909/8952 device", __func__);
     }
@@ -68,7 +70,7 @@ void *hw_info_init(const char *snd_card_name)
     }
 
     if (strstr(snd_card_name, "msm8x16") || strstr(snd_card_name, "msm8909")
-        || strstr(snd_card_name, "msm8952")) {
+        || strstr(snd_card_name, "msm8952") || strstr(snd_card_name, "msm8976")) {
         ALOGV("8x16 - variant soundcard");
 
         strlcpy(hw_info->type, "", sizeof(hw_info->type));
-- 
2.15.1

